README.md

```diff
@@ -112,6 +112,25 @@ docker-compose -f simplewallet-build-tp-<your_prog_language>.yaml up --build
 where,
  <your_prog_language> should be replaced with either `cxx`, `java`, or `py`
 
+# Building and running on OS(without dockers)
+To run sawtooth-simplewallet without dockers, we'll have to use a Ubuntu 16.04 OS installation and compile simplewallet from sources. Below is a sample procedure for python TP/client:
+
+1. Install sawtooth on Ubuntu 16.04 LTS x64 machine. Refer sawtooth app developer's guide [here](https://sawtooth.hyperledger.org/docs/core/releases/latest/app_developers_guide/ubuntu.html)
+2. git clone https://github.com/askmish/sawtooth-simplewallet.git
+3. `cd sawtooth-simplewallet`
```

Comment:
I just tried it and two files need to be modified.  So here are some more instructions:

Add a step:

3A Modify the transaction processor and client as follows:
- `git branch nodocker; git checkout nodocker`
- Edit file `pyclient/wallet/simplewallet_cli.py` and change `rest-api:8008` to `localhost:8008`
- Edit file `pyprocessor/processor/simplewallet_tp.py and change `validator:4004` to `localhost:4004`

Reply:
Ah, I forgot to add these info. Thanks.